### ISSUE TITLE
Work with Node 4.5.x and 4.6.x but not 7.x.x on Mac

### ISSUE THREAD

**Issue by shiqizheng** [etsy/statsd#595]
$node stats.js exampleConfig.js

util.js:965
    throw new TypeError('The super constructor to `inherits` must not ' ...

**Comment by dineshkumar-cse** [comment]
I'm facing the same issue. following is the stacktrace. 

```TypeError: The super constructor to "inherits" must not be null or undefined
    at Object.exports.inherits (util.js:961:11)
    at Object.<anonymous> (.../statsd/lib/config.js:33:6)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (.../statsd/stats.js:4:14)```

**Comment by benburry** [comment]
Resolved by #593

**Comment by ricardobeat** [comment]
This is still the case, no releases since 11 May 2016 so it currently does not work on node > v5.
